std/datetime/date.d

```diff
@@ -12,6 +12,7 @@ module std.datetime.date;
 import core.time;// : TimeException;
 import std.traits : isSomeString, Unqual;
 import std.typecons : Flag;
+import std.range.primitives;// : isOutputRange;
```

Comment:
Please use selective imports for new imports - otherwise you risk leaking the entire module.

Reply:
I thought it was the opposite?

Reply:
We can't convert the existing imports to selective imports because that would close the symbol leakage: https://github.com/dlang/phobos/pull/5584